Sources/AsyncAlgorithms/Buffer/QueuedBufferStorage.swift

```diff
@@ -0,0 +1,106 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Async Algorithms open source project
+//
+// Copyright (c) 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+public enum QueuedBufferPolicy: Sendable {
+  case unbounded
+  case bufferingNewest(limit: UInt)
```

Comment:
usually we use `Int` and fault on invalid values

Reply:
modified.